ext/rbs_extension/parser.c

```diff
@@ -89,34 +87,32 @@ static bool rbs_is_untyped_params(method_params *params) {
 //  * */
 static VALUE rbs_location_current_token(parserstate *state) {
   return rbs_location_pp(
-    state->buffer,
-    &state->current_token.range.start,
-    &state->current_token.range.end
-  );
+      state->buffer,
+      &state->current_token.range.start,
+      &state->current_token.range.end);
 }
 
 static VALUE parse_optional(parserstate *state);
 static VALUE parse_simple(parserstate *state);
 
 static VALUE string_of_loc(parserstate *state, position start, position end) {
   return rb_enc_str_new(
-    RSTRING_PTR(state->lexstate->string) + start.byte_pos,
+      RSTRING_PTR(state->lexstate->string) + start.byte_pos,
       end.byte_pos - start.byte_pos,
-      rb_enc_get(state->lexstate->string)
-  );
+      rb_enc_get(state->lexstate->string));
 }
 
 /**
  * Raises RuntimeError with "Unexpected error " message.
  * */
 static NORETURN(void) rbs_abort(void) {
   rb_raise(
-    rb_eRuntimeError,
-    "Unexpected error"
-  );
+      rb_eRuntimeError,
+      "Unexpected error");
```

Comment:
Could we set `BracketAlignmentStyle: BAS_BlockIndent`

```suggestion
      rb_eRuntimeError,
      "Unexpected error"
  );
```

Reply:
I think it's `AlignAfterOpenBracket: BlockIndent` instead. Updated